src/internal/connector/onedrive/common/consts.go

```diff
@@ -0,0 +1,13 @@
+package common
```

Comment:
I'd like us to minimize the notion of a "common" package: the naming isn't helpful, the packages tend to bloat, and that all results in collisions (see: `odcommon` aliased imports).  Thoughts on moving this into `internal/common/metadata` instead?  The functionality will overextend onedrive soon (sharepoint), and potentially again in the future.  Plus, that move would jive well with the other suggestion for moving the logic out of graph, too.

Reply:
I'm not really sure it belongs outside of the OneDrive tree though. No other service outside of OneDrive stuff really has the notion of metadata. And really nothing besides OneDrive and tests need this. We have it in details and selectors as a hack to work around how things are now or some old ways of doing things, but I'm not really sure yet if we have a stable idea of what "metadata" is or isn't

Reply:
That's all fine, I'm not terribly concerned with the location, so much as being nitpicky about naming and not spreading logic between two packages.  `/connector/onedrive/metadata`, maybe?